### PR TITLE
Clarify tiling of AtlasTexture

### LINKED ISSUE #5660: AtlasTexture does not repeated in sprites
**Operating system or device - Godot version:**
Godot Engine 2.0.4.1 (Ubuntu 16.04 - 64bits)

**Issue description** (what happened, and what was expected):
The AtlasTexture is not repeated on the sprite.

**Steps to reproduce:**
* Create a Sprite
* Load the AtlasTexture (or create a new one)
* Check Region on and set Region Rect (0, 0, 300, 300) (to have the "repeated effect")
* Click Edit the Texture
* Click Edit the Atlas
* Check Repeat on ImageTexture Flags.

**Image example:**

sprite_atlastexture_bug

### PR DESCRIPTION
**Pull request by KoBeWi**
Closes #5660

### CODE REVIEW

**Review by AThousandShips — APPROVED** [review]

**Comment by arkology** [comment]
Is it a documentation issue if tiling worked before (according to https://github.com/godotengine/godot/issues/5660#issuecomment-1257704120)?

**Comment by KoBeWi** [comment]
Seems like it works if you set region of the AtlasTexture itself, not Sprite2D or TextureRect.

**Comment by KoBeWi** [comment]
Pushed additional note.

File: doc/classes/AtlasTexture.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<description>
 		[Texture2D] resource that draws only part of its [member atlas] texture, as defined by the [member region]. An additional [member margin] can also be set, which is useful for small adjustments.
 		Multiple [AtlasTexture] resources can be cropped from the same [member atlas]. Packing many smaller textures into a singular large texture helps to optimize video memory costs and render calls.
-		[b]Note:[/b] [AtlasTexture] cannot be used in an [AnimatedTexture], and may not tile properly in nodes such as [TextureRect], when inside other [AtlasTexture] resources.
+		[b]Note:[/b] [AtlasTexture] cannot be used in an [AnimatedTexture], and will not tile properly in nodes such as [TextureRect] or [Sprite2D]. For tiling an [AtlasTexture], modify its own [member region] instead.
```
Comment (Mickeon) [inline doc/classes/AtlasTexture.xml:9]:
```suggestion
		[b]Note:[/b] [AtlasTexture] cannot be used in an [AnimatedTexture], and will not tile properly in nodes such as [TextureRect] or [Sprite2D]. To tile an [AtlasTexture], modify its [member region] instead.
```

**Review by Mickeon — APPROVED** [review]

**Comment by Repiteo** [comment]
Thanks!

**Comment by Mickeon** [comment]
Would we rather revert this and/or not recommend anything?

**Comment by KoBeWi** [comment]
I'd just remove the last sentence, the other part is correct.
There are supposedly some workarounds, like using shaders, but it's probably too complex to explain here.

**Pull request merged**